app/redux/reducers/SettingsReducer.js

```diff
@@ -22,10 +28,28 @@ const defaultState = {
     wasMetricEdited: false,
     showRemoved: false,
     isExportingCSV: false,
-    lastExportCSVDate: null,    
+    lastExportCSVDate: null,
+    metric1: 'Velocity',
+    quantifier1: 'Last Rep',
+    metric2: 'PKV',
+    quantifier2: 'Last Rep',
+    metric3: 'PKH',
+    quantifier3: 'Last Rep',
+    metric4: 'ROM',
+    quantifier4: 'Last Rep',
+    metric5: 'Duration',
+    quantifier5: 'Last Rep',
+    isEditingMetric: false,
+    isEditingAvgMetric: false,
+    isEditingBestEverMetric: false,
```

Comment:
- [ ] Instead of using these three booleans, just pass in the quantifier type to the screen and have the screen define booleans for which metrics to actually display